cmd/titus-metadata-service/main.go

```diff
@@ -91,6 +93,30 @@ func readTaskConfigFile(taskID string) (*titus.ContainerInfo, error) {
 	return &cInfo, nil
 }
 
+func readTaskPodFile(taskID string) (*corev1.Pod, error) {
+	if taskID == "" {
+		log.Errorf("task ID is empty: can't read pod config file")
+		return nil, fmt.Errorf("task ID env var unset: %s", taskInstanceIDEnvVar)
+	}
+
+	// This filename is from VK, which is /run/titus-executor/$namespace__$podname/pod.json
+	// We only use the default namespace, so we hardcode it here.
+	confFile := filepath.Join("/run/titus-executor/default__", taskID, "pod.json")
```

Comment:
It would be kinda nice if this path was abstracted behind a function somehow, or a constant somewhere - wouldn't this also be used in the titus-vk implementation?

Reply:
I agree.

There *is* some code in VK that does this calculation. The only weirdness is that it assumes you *already* have the pod object. Here we only know the taskid. Eh.

@sargun would it be ok to import executorapi in here from VK? Or would it be better just to hard code this path?